### PR TITLE
Write basic dev setup directions in README & set up database seeding

### PR DESCRIPTION
**Pull request by ybur-yug**
Currently, we lack any documentation on how to get the project running in development. And, if one were to, they would be greeted with an empty page. This alleviates that issue by simple adding a basic database seed and documenting the setup process.

I have run this locally with no database and off of a fresh clone and it seemed to work perfectly on my setup. I'd like for someone else to verify.

This is what the homepage looks like after seeding + building JS and running the server, now.

homepage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,45 @@
...
+Now we seed the database
+
+```bash
+$ mix run apps/data/priv/repo/seeds.exs
+```
+
+From here, we need to build all of our javascript pieces.
+
+```bash
+$ cd apps/web/assets
```
Comment (doomspork) [inline README.md]: I'm not sure these are accurate or necessary, are they?  I ran the project earlier and `phx.server` took care of compiling my assets but I think @burden set it up to work with webpack and not brunch.

File: apps/data/priv/repo/seeds.exs [hunk 2]
```diff
@@ -9,3 +9,50 @@
...
+  tags: ["elixir", "open source", "blog"],
+  url: "https://github.com/elixirschool/appendix",
+}
+
+{:ok, elixir_school}   = Data.Projects.insert(project_params_1)
+{:ok, extracirricular} = Data.Projects.insert(project_params_2)
+{:ok, appendix}        = Data.Projects.insert(project_params_3)
+
+opportunity_params_1 = %{
+  title: "Blog post on Git Bisect",
```
Comment (doomspork) [inline apps/data/priv/repo/seeds.exs]: How about we use something that's likely to be covered by an Elixir blog instead?

"Understanding GenServers and State"

File: apps/data/priv/repo/seeds.exs [hunk 3]
```diff
@@ -9,3 +9,50 @@
...
+
+opportunity_params_2 = %{
+  title: "Translate to pig latin",
+  level: 1,
+  url: "https://github.com/elixirschool/elixirschool/issues/1",
+  project_id: elixir_school.id
+}
+
+opportunity_params_3 = %{
+  title: "Rewrite it in Node/Ember lol",
```
Comment (doomspork) [inline apps/data/priv/repo/seeds.exs]: Let's stick to realistic seed data:  "Implement API for sorting" or something like that

File: README.md (hunk 1 above)
Comment (ybur-yug) [reply README.md]: Oh I've never used webpack but when I did a normal run of `phx.server` without having done this it wouldnt work, and brunch wouldnt run until I had `cd`'d here.

**Pull request merged**